docker/rucio-daemons-hermesk/Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG RUCIO_DAEMONS_VERSION
+FROM rucio/rucio-daemons:${RUCIO_DAEMONS_VERSION}
+
+ARG CONFLUENT_KAFKA_VERSION
+RUN pip install confluent_kafka==${CONFLUENT_KAFKA_VERSION}
+RUN pip install retrying
+
+RUN mkdir -p /etc/grid-security/certificates
+RUN mkdir /usr/local/lib/python3.9/site-packages/rucio/daemons/hermes/kafka
+COPY ./lib/rucio/daemons/hermes/kafka /usr/local/lib/python3.9/site-packages/rucio/daemons/hermes/kafka/
```

Comment:
It's better to copy the slowly-changing things first, then the rapidly-changing things.  That way the layer cache works better.

Reply:
Done